doc/requirements.txt

```diff
@@ -1,11 +1,11 @@
-doc8>=0.8.0
```

Comment:
shouldn't theses requirement files be removed now?

Reply:
Emm yes... not sure what I was doing there...

Reply:
Done: dc2ee290322fd32b36f59929706e47c66cf19c22